XenAdmin/Alerts/Types/ClientUpdateAlert.cs

```diff
@@ -153,40 +157,30 @@ public static void DownloadSource(IWin32Window parent)
             // If no update no need to show where to save dialog
             var clientVersion = Updates.ClientVersions.FirstOrDefault();
 
-            DownloadSourceAction downloadSourceAction;
             if (clientVersion == null)
             {
-                // There is no XCupdates.xml so direct to website.
+                // There is no XCUpdates.xml so direct to website.
                 Program.OpenURL(InvisibleMessages.WEBSITE_DOWNLOADS);
-                return;
             }
             else
             {
                 string outputPathAndFileName;
                 using (var saveSourceDialog = new SaveFileDialog())
                 {
-                    saveSourceDialog.FileName = string.Format(Messages.SOURCE_FILE_NAME, BrandManager.BrandConsole, clientVersion.Version) + ".zip";
+                    saveSourceDialog.FileName = $"{BrandManager.BrandConsole}-v{clientVersion.Version}-source.zip";
                     saveSourceDialog.DefaultExt = "zip";
                     saveSourceDialog.Filter = "(*.zip)|*.zip|All files (*.*)|*.*";
                     saveSourceDialog.InitialDirectory = Win32.GetKnownFolderPath(Win32.KnownFolders.Downloads);
 
-                    if (saveSourceDialog.ShowDialog() != DialogResult.OK)
+                    if (saveSourceDialog.ShowDialog(parent) != DialogResult.OK)
                     {
                         return;
                     }
                     outputPathAndFileName = saveSourceDialog.FileName;
                 }
 
-                downloadSourceAction = new DownloadSourceAction(clientVersion.Name, clientVersion.Version, new Uri(clientVersion.SourceUrl), outputPathAndFileName);
-
-                using (var dlg = new ActionProgressDialog(downloadSourceAction, ProgressBarStyle.Continuous))
-                {
-                    dlg.ShowCancel = true;
-                    dlg.ShowDialog(parent);
-                }
-
-                if (!downloadSourceAction.Succeeded)

```

Comment:
This check is not needed since we don't do anything in case od success; the method will return anyway.